pkg/plugins/log/logger.go

```diff
@@ -23,7 +23,6 @@ func (d *grafanaInfraLogWrapper) New(ctx ...any) Logger {
 		}
 	}
 
-	ctx = append([]any{"logger"}, ctx...)
```

Comment:
@wbrowne seems like you added this logger argument. Removed that now since it resulted in LOG15 error in logs. As far as I can tell the logger argument shouldn't be there since the logger you're creating a new logger from should already have the logger name set.

Reply:
Hmmm... Don't remember details on this specifically. I think I based these changes off of https://github.com/grafana/grafana/blob/main/pkg/infra/log/log.go#L239-L246

Reply:
Ohh I see so that's when you create a completely new logger and description says:
> // First ctx argument is expected to be the name of the logger.

For the concrete logger as in our case here we don't want to have this logic
https://github.com/grafana/grafana/blob/534ca297072f89dd28c5c9b3024f136a79658b12/pkg/infra/log/log.go#L219
https://github.com/grafana/grafana/blob/534ca297072f89dd28c5c9b3024f136a79658b12/pkg/infra/log/interface.go#L16-L17

So should be fine removing this logic :+1: